java/ql/test/experimental/stubs/springframework-5.2.3/org/springframework/web/bind/annotation/RequestParam.java

```diff
@@ -0,0 +1,8 @@
+package org.springframework.web.bind.annotation;
```

Comment:
Reusing the existing spring stubs means that this file isn't needed (it's already in `java/ql/test/stubs/springframework-5.2.3`)

Reply:
Removed.